packages/-ember-data/testem.js

```diff
@@ -1,16 +1,34 @@
 const customDotReporter = require('@ember-data/unpublished-test-infra/src/testem/custom-dot-reporter');
 
 const TestIE = process.env.TEST_IE11;
+const TestSafari = process.env.TEST_SAFARI;
 
 if (TestIE) {
   // eslint-disable-next-line no-console
   console.log('\n\nLaunching with IE\n\n');
 }
+if (TestSafari) {
+  // eslint-disable-next-line no-console
+  console.log('\n\nLaunching with Safari\n\n');
+}
+
+function browserPicker() {
```

Comment:
To avoid the function and the duplication across testem.js files we could change this from boolean process flags to a string.

```
const BrowserLauncher = process.env.TESTEM_CI_LAUNCHER || 'Chrome'; 

...


launch_in_ci: [BrowserLauncher],

Reply:
The log could be

```
console.log(`\n\nLaunching with ${BrowserLauncher}\n\n`);
```